models/kyberswap/avalanche_c/kyberswap_avalanche_c_trades.sql

```diff
@@ -30,8 +31,8 @@ kyberswap_dex AS (
         ,t.evt_tx_hash                                                      AS tx_hash
         ,''                                                                 AS trace_address
         ,t.evt_index
-    FROM {{ source('kyber_avalanche_c', 'DMMPool_evt_Swap') }} t
-    INNER JOIN {{ source('kyber_avalanche_c', 'DMMFactory_evt_PoolCreated') }} p
+    FROM {{ source('kyber_avalanche_c_trades', 'DMMPool_evt_Swap') }} t
+    INNER JOIN {{ source('kyber_avalanche_c_trades', 'DMMFactory_evt_PoolCreated') }} p
```

Comment:
In order to make the compiled spell work, we need to change back every `kyber_avalanche_c_trades` to `kyber_avalanche_c` in this file.